jobs/Scripts/simpleRender.py

```diff
@@ -88,16 +88,24 @@ def createArgsParser():
 	return parser
 
 
-def check_licenses(res_path, maya_scenes):
+def check_licenses(res_path, maya_scenes, testType):
 	try:
 		for scene in maya_scenes:
-			with open(os.path.join(res_path, scene[:-1])) as f:
+			try:
+				scenePath = os.path.join(res_path, testType, scene[:-4])
+				if not os.path.isdir(scenePath):
+					scenePath = os.path.join(res_path, testType)
+				scenePath = os.path.join(scenePath, scene[:-1])
+			except:
+				scenePath = os.path.join(res_path, testType, scene[:-1])
+
+			with open(scenePath) as f:
 				scene_file = f.read()
 
 			license = "fileInfo \"license\" \"student\";"
```

Comment:
Просто взгляд зацепился.
```suggestion
			license = 'fileInfo "license" "student";'
```